README.md

```diff
@@ -1 +1,51 @@
-# angular-custom-validation-readme
\ No newline at end of file
+# Custom validation with $validators
+
+## Overview
+
+The built-in validation in HTML forms can only go so far. However, we can actually define custom validation rules!
+
+## Objectives
+
+- Describe the $validators pipeline
+- Write a custom validator
+- Integrate the custom validator with ngMessages
+
+## $validators
+
+Much like `$parsers` and `$formatters`, we add our custom validators into `$validators`. Here we have a function where we can test if the value is valid, returning true or false depending on their validity.
+
+Let's create a directive to see if our input is *only* numbers.
```

Comment:
We haven't really seen directives used like this before - add an explanation either to this README or the previous one about how we can use directives to simply test or replace content.